lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -93,3 +97,45 @@ def fit_transform(self, x):
 
     def inverse_transform(self, y):
         return self.scaler.inverse_transform(y)
+
+
+class DateNormalizer:
```

Comment:
Would it not be possible to share this with the current date encoder ?

Reply:
Do you mean as in adding these functionalities to the encoder class instead?

Reply:
Okay, functionality is migrated into the date encoder, looks better now.